tests/installation/partitioning_raid.pm

```diff
@@ -528,7 +526,7 @@ sub enter_partitioning {
 
 sub run {
     # for newer storage-ng toolbar has changed
-    $cmd{addraid} = 'alt-d' unless $older_product;
+    $cmd{addraid} = 'alt-d' if is_new_storage_ng;
```

Comment:
LGTM all the changes, I would suggest to add this in partitioning.pm and delete the comment above so we don't have this orphan line.